odbc/src/api/functions.rs

```diff
@@ -254,6 +255,8 @@ fn sql_alloc_handle(
     input_handle: *mut MongoHandle,
     output_handle: *mut Handle,
 ) -> Result<()> {
+    load_mongosqltranslate_library();
```

Comment:
This seemed like a good spot to load it, since the allocating of the handle was usually the first operation.